### DISCUSSION TITLE
Looking for help with package installation

### DISCUSSION THREAD

**Discussion by CharlyJazz** [orchetect/swift-midi#154] [Q&A] [answered]
I installed the package and copy paste this [example](https://orchetect.github.io/MIDIKit/documentation/midikit/simple-midi-listener-class-example).

But Xcode raise this error: 
<img width="775" alt="image" src="https://user-images.githubusercontent.com/12489333/210415394-45a619c6-5b2f-420d-843c-c25f5c7ed057.png">

I am using a Mac M1 Laptop

Sorry if I am such a big noob, I am new in swift!

**Comment by orchetect** [comment, marked as answer]
Make sure you add MIDIKit as a dependency to your project then ensure to link the library with your project if it isn’t already.

Every Swift Package you use will require that you do that.

There are also lots of complete example projects in the Examples folder you can check out.

**Reply by CharlyJazz** [reply]
Thanks!
